packages/flutter_tools/test/general.shard/update_packages_test.dart

```diff
@@ -42,7 +42,6 @@ dev_dependencies:
     sdk: flutter
   flutter_goldens:
     sdk: flutter
-  mockito: 4.1.1
```

Comment:
This doesn't really serve any purpose so lets remove it to prevent it from showing up in my searches